mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterGatewayAkkaImpl.java

```diff
@@ -49,13 +57,26 @@ class ResourceClusterGatewayAkkaImpl implements ResourceClusterGateway {
         this.mapper = mapper;
 
         this.rateLimiter = RateLimiter.create(maxConcurrentRequestCount);
+        Metrics m = new Metrics.Builder()
+                .id("ResourceClusterGatewayAkkaImpl")
+                .addCounter("registrationCounter")
+                .addCounter("heartbeatCounter")
+                .addCounter("disconnectionCounter")
+                .addCounter("throttledCounter")
+                .build();
+        Metrics metrics = MetricsRegistry.getInstance().registerAndGet(m);
+        this.registrationCounter = metrics.getCounter("registrationCounter");
```

Comment:
is it possible to add the RC tag?

Reply:
the metrics here are static without dynamic tags, which should be fine as we can track detailed operations at rc actor level. I am trying to avoid the dynamic metrics id in these large volume ones.

Reply:
Doesn't Akka or our integration with Akka automatically capture calls per method?